pkg/cmd/tenant/create/tenant_create.go

```diff
@@ -216,7 +217,10 @@ func createTenant(
 	t *coretnt.Tenant,
 	parentTenant *coretnt.Tenant,
 ) (tenant.CreateOrUpdateResult, error) {
-	spinnerHandler := streams.Spinner("Creating tenant...")
+	spinnerHandler := streams.Wizard(
+		fmt.Sprintf("Creating tenant: %s", t.Name),
```

Comment:
include the where you are creating tenant ? (url)

Reply:
Added